halyard-deploy/src/main/resources/debian/init.sh

```diff
@@ -1,23 +1,4 @@
-source /etc/os-release
-
-if [ "$VERSION_ID" = "14.04" ]; then
```

Comment:
Removed since this is for ancient distros that are out of support, only Ubuntu 18.04 LTS onwards still has support.